util.js

```diff
@@ -40,3 +40,15 @@ exports.getAvaConfig = filepath => {
 exports.isFunctionExpression = node => {
 	return node && functionExpressions.indexOf(node.type) !== -1;
 };
+
+exports.getTestModifiers = function getTestModifiers(node) {
```

Comment:
Doesn't work, as getTestModifiers is called recursively. It needs to either be declared before and then re-assigned to exports, or declared as a function expression, which I prefer in this case.